server/api/__snapshots__/team.test.js.snap

```diff
@@ -40,15 +41,16 @@ Object {
 
 exports[`#team.removeAdmin should require admin 1`] = `
 Object {
-  "error": "authorization_error",
-  "message": "Authorization Required",
+  "error": "admin_required",
+  "message": "An admin role is required to access this resource",
   "ok": false,
+  "status": 403,
 }
 `;
 
 exports[`#team.removeAdmin shouldn't demote admins if only one available  1`] = `
 Object {
-  "error": "at_least_one_admin_is_required",
+  "error": "validation_error",
```

Comment:
This is an example if we replace `httpErrors.BadRequest` out in all the spots it's used - the error responses are more useable. I'll leave that for a separate PR though.